zentral/contrib/osquery/serializers.py

```diff
@@ -220,3 +221,37 @@ def update(self, instance, validated_data):
         if compliance_check_deleted:
             query.refresh_from_db()
         return query
+
+
+class ConfigurationPackSerializer(serializers.ModelSerializer):
+    configuration = serializers.PrimaryKeyRelatedField(queryset=Configuration.objects.all())
+
+    class Meta:
+        model = ConfigurationPack
+        fields = "__all__"
+
+    def validate_configuration(self, value):
```

Comment:
Multiple packs can be added to the configuration. You should test that this is possible. This validation method is probably not necessary.